xtrlock.man

```diff
@@ -48,3 +48,15 @@ This program is distributed in the hope that it will be useful, but
 WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 General Public License for more details.
+
+This program now allows user to use their own password instead of 
+the default user password, which is the sudo password (for many 
+desktop users, I know that!) that one may not want to reveal. Using 
+another password and share it with your friend could solved this. 
```

Comment:
I'd like to check out what's inside your script. However, the domain name can't be resolved from both my computer(Using a DNS of 8.8.8.8) and my friends' in Australia. Since this domain name looks like yours, you may want to double-check if it's working normally.